django/cantusdb_project/main_app/models/chant.py

```diff
@@ -1,5 +1,6 @@
 from django.db.models.query import QuerySet
 from main_app.models.base_chant import BaseChant
+from align_text_mel import *
```

Comment:
I think it would be better to be specific here -- are there lots of functions you are using?